internal/btesting/btesting.go

```diff
@@ -54,6 +54,7 @@ func MustOpenDBWithOption(t testing.TB, f string, o *bolt.Options) *DB {
 
 	db, err := bolt.Open(f, 0666, o)
 	require.NoError(t, err)
+	db.StrictMode = true
```

Comment:
Long term probably should be an ENV_VARIABLE driven. Test has it's default but if env variable is defined, it takes precedence.

Reply:
agreed

Reply:
FYI. https://github.com/etcd-io/bbolt/pull/440